staging/src/k8s.io/apiserver/pkg/server/options/etcd_test.go

```diff
@@ -53,7 +53,7 @@ func TestEtcdOptionsValidate(t *testing.T) {
 				EnableGarbageCollection: true,
 				EnableWatchCache:        true,
 				DefaultWatchCacheSize:   100,
-				EtcdServersOverrides:    []string{"/events#http://127.0.0.1:4002"},
+				EtcdServersOverrides:    []string{"/events#http://127.0.0.1:4002#/registry"},
```

Comment:
leave existing tests as-is to ensure backwards compatibility, and add new tests to exercise the new syntax